recipes/jbig/Makefile.patch

```diff
@@ -0,0 +1,11 @@
+--- Makefile	2014-03-27 14:47:15.000000000 -0400
++++ Makefile	2015-11-06 14:14:48.000000000 -0500
+@@ -4,7 +4,7 @@
+ CC = gcc
+ 
+ # Options for the compiler: A high optimization level is suggested
+-CFLAGS = -O2 -W -Wno-unused-result
++CFLAGS = -O2 -W -fPIC
```

Comment:
So, the `sed` command takes care of part of this. Why is it we `-fPIC` here? Couldn't we just set the `CFLAGS` in advance?